Podcast/UserSettings.swift

```diff
@@ -9,8 +9,8 @@
 import UIKit
 
 class UserSettings: NSObject {
-    
-    static let sharedSettings = UserSettings()
+
+    static var sharedSettings = UserSettings()
```

Comment:
shared objects should not be var as they shouldn't be changing...

Reply:
changed